### ISSUE TITLE
LINQ support

### ISSUE THREAD

**Issue by ygorlazaro** [spring-projects/spring-net#127]
Is there any idea/plan to make LINQ support native at spring?

Better: is spring.net still under development?

**Comment by gabrielsimas** [comment]
Yes. My Friend. It may not seem, but is still in development but
